microsoft-edge/extensions-chromium/store-policies/csp.md

```diff
@@ -29,15 +29,22 @@ On the web, such a policy is defined via an HTTP header or `meta` element.  Insi
 <!-- ====================================================================== -->
 ## Default Policy Restrictions
 
-Packages that don't define a `manifest_version` don't have a default content security policy.  Packages that use `manifest_version` 2 have the following default content security policy:
+Packages that don't define a `manifest_version` don't have a default content security policy.
+
+<!-- todo: state the situation for Manifest 3:
+Packages that use `manifest_version` 3 have ...
+-->
```

Comment:
PM/Dev:
state the situation for Manifest 3:
Packages that use `manifest_version` 3 have ...